fluent-plugin-json-in-json.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "yajl-ruby", "~> 1.0"
+  spec.add_runtime_dependency "fluentd", '>= 0.14.0'
```

Comment:
`spec.add_runtime_dependency "fluentd", ['>= 0.14.0', '< 2']` would be better.

ref: https://docs.fluentd.org/v0.14/articles/plugin-update-from-v12#2.-update-dependency